diff --git a/.changeset/wise-toes-joke.md b/.changeset/wise-toes-joke.md
new file mode 100644
index 0000000..c685b7a
--- /dev/null
+++ b/.changeset/wise-toes-joke.md
@@ -0,0 +1,5 @@
+---
+"@preact/signals-react": patch
+---
+
+Fixed typescript error from resolving unspecified in exports map .ts file
diff --git a/packages/react/src/index.ts b/packages/react/src/index.ts
index 9c47acb..c5323fc 100644
--- a/packages/react/src/index.ts
+++ b/packages/react/src/index.ts
@@ -13,7 +13,7 @@ import {
 	useComputed,
 	useSignalEffect,
 	installAutoSignalTracking,
-} from "../runtime/src/index"; // TODO: This duplicates runtime code between main and sub runtime packages
+} from "@preact/signals-react/runtime"; // TODO: This duplicates runtime code between main and sub runtime packages
 
 export {
 	signal,
